airflow/providers/openlineage/extractors/base.py

```diff
@@ -82,7 +82,7 @@ def extract(self) -> OperatorLineage | None:
             self.operator.__class__.__module__ + "." + self.operator.__class__.__name__
         )
         if fully_qualified_class_name in self.disabled_operators:
-            self.log.warning(
+            self.log.info(
```

Comment:
This should be DEBUG too, no? i.e user performed an action to disable the lineage for specific operators

Reply:
Good catch.